nilearn/datasets/atlas.py

```diff
@@ -2710,15 +2609,54 @@ def fetch_atlas_schaefer_2018(
     )
     lut = _update_lut_freesurder(lut)
 
-    labels = list(lut["name"])
-    fdescr = get_dataset_descr(dataset_name)
-
-    _check_look_up_table(lut=lut, atlas=atlas_file)
-
-    return Bunch(
+    return Atlas(
         maps=atlas_file,
-        labels=labels,
-        description=fdescr,
+        labels=list(lut["name"]),
+        description=get_dataset_descr(dataset_name),
         lut=lut,
         atlas_type=atlas_type,
     )
+
+
+class Atlas(Bunch):
+    """Sub class of Bunch to help standardize atlases."""
+
+    def __init__(
+        self,
+        maps,
+        description,
+        atlas_type,
+        labels=None,
+        lut=None,
+        template=None,
+        **kwargs,
+    ):
+        assert atlas_type in ["probabilistic", "deterministic"]
+
+        # TODO: improve
+        if template is None:
+            template = "volume"
```

Comment:
- This lays the ground to fix https://github.com/nilearn/nilearn/issues/3075

Reply:
at the moment I have only taken care of the easy and obvious ones: fsaverage and talairach

figuring out which atlas in which MNI space is the kind of rabbit hole that I am looking forward to explore and that deserves its own PR